crates/typst/src/text/mod.rs

```diff
@@ -37,6 +37,7 @@ use ttf_parser::{Rect, Tag};
 
 use crate::diag::{bail, warning, HintedStrResult, SourceResult};
 use crate::engine::Engine;
+use crate::foundations::IntoValue;
```

Comment:
please move this import into the list below (I wish rustfmt's import settings were stable :/)